conductor/lib/maya_utils.py

```diff
@@ -472,30 +472,32 @@ def get_render_layers_info():
     return render_layers
 
 
-def collect_dependencies(node_attrs, leaf_path_list=None, exclude_path_list=None):
+def collect_dependencies(node_attrs, leaf_path=None, exclude_path=None):
```

Comment:
shouldn't these variable names be pluralized... since they are indeed a list of paths?   e.g. `leaf_paths` and `exclude_paths`?

Reply:
Yup - I'll fix again ;)